client-desktop/src/app/components/Editor/index.js

```diff
@@ -35,7 +38,12 @@ class Editor extends Component {
 
   componentDidMount() {
     client.watchQuery({ query: GET_CURRENT_NOTE })
-    .subscribe(({data}) => this.setState({ value: html.deserialize(data.currentNote.body)}))
+    .subscribe(({data}) => {
+      data ?
+      this.setState({ value: html.deserialize(data.currentNote.body)})
+      :
+      this.setState({ value: html.deserialize('') })
```

Comment:
Just a suggestion, a long time ago I saw this style for ternaries that I really like: 

```js
data 
    ? this.setState({ value: html.deserialize(data.currentNote.body)})
    : this.setState({ value: html.deserialize('') })
```